src/components/calculator.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Buttons from './buttons';
+import Display from './display';
+import calculate from '../logic/calculate';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
```

Comment:
- [ ] We shall kindly make sure that the Calculator component is functional-based. **Hints**


```jsx
function Calculator() {
  return <div>{/* Calculator app design goes here */}</div>
}
```